### PR TITLE
Updating help text and error message for choosing username on sign up page

### PR DESCRIPTION
**Pull request by lisakycho**
#### Summary
Help text and error message for choosing username is updated when a new user is signing up to the following:
Help text: "You can use lowercase letters, numbers, periods, dashes, and underscores."
Error message for username that does not meet requirements: "Usernames have to begin with a lowercase letter and be 3-22 characters long. You can use lowercase letters, numbers, periods, dashes, and underscores."

image
image

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/10552

### CODE REVIEW

**Comment by hanzei** [comment]
@lisakycho Thanks for this PR! :rocket: 

Would you please you single quotes for the `defaultMessage`? This will fix the build error. Then I can set up a test server.

**Comment by lisakycho** [comment]
@hanzei, I updated the pull request! :)

File: i18n/en.json [hunk 1]
```diff
@@ -2805,8 +2805,8 @@
   "signup_user_completed.required": "This field is required",
   "signup_user_completed.reserved": "This username is reserved, please choose a new one.",
   "signup_user_completed.signIn": "Click here to sign in.",
-  "signup_user_completed.userHelp": "Username must begin with a letter, and contain between {min} to {max} lowercase characters made up of numbers, letters, and the symbols '.', '-' and '_'",
-  "signup_user_completed.usernameLength": "Username must begin with a letter, and contain between {min} to {max} lowercase characters made up of numbers, letters, and the symbols '.', '-' and '_'.",
+  "signup_user_completed.userHelp": "You can use lowercase letters, numbers, periods, dashes, and underscores",
```
Comment (lieut-data) [inline i18n/en.json]: Should this message end with a trailing `.`?

Comment (hanzei) [reply i18n/en.json]: @esethna What do you think about this?

Comment (lieut-data) [reply i18n/en.json]: @esethna: ^ ping

Comment (esethna) [reply i18n/en.json]: Yes it should contain a period. Sorry missed the notifications here, thanks for the ping

**Comment by lisakycho** [comment]
@esethna, @lieut-data fixed and good to go! :)

**Pull request merged**